owners/cron.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019, the AMP HTML authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cron:
+# Note that both of these happen automatically in response to webhooks for new
+# or edited owners file and teams; this cron task ensures that the app stays in
+# sync even if webhooks are dropped or fail.
+
+- description: "Sync with the latest owners file and re-parse the owners tree"
+  url: /_cron/refreshTree
+  schedule: every hour from 7:00 to 23:00
```

Comment:
We're technically an international project, might as well make these 24/7 :)

Reply:
good call!

Reply:
Done